ci-operator/config/rh-ecosystem-edge/kernel-module-management/rh-ecosystem-edge-kernel-module-management-main.yaml

```diff
@@ -41,6 +41,14 @@ images:
     RUN rm -rf /etc/yum.repos.d/*
     RUN curl http://base-4-13-rhel9.ocp.svc > /etc/yum.repos.d/base-4-13-rhel9.repo
   to: ubi9-minimal-entitled
+- dockerfile_literal: |
+    FROM src
+    # Installing the `oc` client.
+    RUN curl --retry 3 -SL https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/stable/openshift-client-linux.tar.gz \
+        | tar -xz -C /usr/local/bin
+    RUN pip3 install yq
```

Comment:
Shall we use the [Go `yq`](https://github.com/mikefarah/yq) instead? It has 3 to 4 times the stars on GitHub, and is available as a single binary.

Reply:
Sure. Replaced it with the Go yq.